proposed/2020/Transitive-Dependencies.md

```diff
@@ -78,7 +78,7 @@ Finally, selecting a transitive dependency will provide the user the ability to
 
 #### Detailed design
 
-**Collapsible, vertical list group:** Direct and transitive packages will be grouped under collapsible list group
+**Collapsible, vertical list group:** Direct and transitive packages will be grouped under collapsible list group (both expanded by default).
```

Comment:
Can you also update the following:
https://github.com/NuGet/Home/blob/d180ca7b610156a9acbc45f57ad777eaf4a78431/proposed/2020/Transitive-Dependencies.md?plain=1#L71

https://github.com/NuGet/Home/blob/d180ca7b610156a9acbc45f57ad777eaf4a78431/proposed/2020/Transitive-Dependencies.md?plain=1#L85

https://github.com/NuGet/Home/blob/d180ca7b610156a9acbc45f57ad777eaf4a78431/proposed/2020/Transitive-Dependencies.md?plain=1#L94

Reply:
Done 👍